packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -199,6 +190,9 @@ export const UpdateProjectConnection: FC<{
             warehouse: {
                 ...data?.warehouseConnection,
             },
+            dbtVersion: data?.dbtVersion
+                ? (data?.dbtVersion as SupportedDbtVersions)
```

Comment:
why do we need to force the type here? isn't it recognizing the SupportedDbtVersions from the project type ?